cabot/cabotapp/defs.py

```diff
@@ -0,0 +1,27 @@
+# Constant definitions
+
+CHECK_TYPES = (
+    ('>', 'Greater than'),
+    ('>=', 'Greater than or equal'),
+    ('<', 'Less than'),
+    ('<=', 'Less than or equal'),
+    ('==', 'Equal to'),
+)
+
+RAW_DATA_LIMIT = 500000
+
+DEFAULT_CHECK_FREQUENCY = 5
+DEFAULT_CHECK_RETRIES   = 0
+
+DEFAULT_GRAPHITE_EXPECTED_NUM_HOSTS   = 0
+DEFAULT_GRAPHITE_EXPECTED_NUM_METRICS = 0
+DEFAULT_GRAPHITE_INTERVAL             = 5
+
+DEFAULT_HTTP_TIMEOUT     = 30
+MAX_HTTP_TIMEOUT         = 32
+DEFAULT_HTTP_STATUS_CODE = 200
+
+DEFAULT_TCP_TIMEOUT = 8
+MAX_TCP_TIMEOUT     = 16
+
+METRIC_STATUS_TIME_RANGE_DEFAULT = 30
```

Comment:
nit: there's a separate defs.py in metricsapp/, let's put metrics stuff there (also then we can remove the `METRICS_STATUS` prefix)